docs/src/main/sphinx/release/release-411.md

```diff
@@ -33,27 +59,80 @@
+* Add support for coercion between `char` types of different lengths. ({issue}`16402`)
 * Improve performance of queries with joins where both sides of a join have keys
   with the same table bucketing definition. ({issue}`16381`)
 * Improve query planning performance for queries scanning tables with a large
   number of columns. ({issue}`16203`)
+* Improve scan performance for `COUNT(*)` queries on row-oriented formats. ({issue}`16595`)
+* Ensure the value of the `hive.metastore-stats-cache-ttl` configuration
+  property always is greater than or equal to the value specified in the
+  `hive.metastore-cache-ttl` configuration property. ({issue}`16625`)
+* Skip listing Glue metastore tables with invalid column types. ({issue}`16677`)
 * Fix query failure when a file that is using a text file format with a
   single header row is split. ({issue}`16492`)
-* Fix query failure when reading text files that have been replaced during the
-  query. ({issue}`16510`)
+* Fix potential query failure with a Kerberized Hive connector when the query
+  executes longer than a Kerberos ticket's lifetime. ({issue}`16680`)
+
+## Hudi connector
+
+* Add a `$timeline` system table which can be queried to inspect the Hudi table
+  timeline. ({issue}`16149`)
 
 ## Iceberg connector
 
-* Add support for a `migrate` procedure that converts a Hive table to an Iceberg
-  table. ({issue}`13196`)
-* Add support for materialized views with a defined grace period. ({issue}`15842`)
+* Add a `migrate` procedure that converts a Hive table to an Iceberg table. ({issue}`13196`)
+* Add support for materialized views with a freshness grace period. ({issue}`15842`)
 * Add a `$refs` system table which can be queried to inspect snapshot
   references. ({issue}`15649`)
+* Add support for creation of materialized views partitioned with a temporal
+  partitioning function on a `timestamp with time zone` column. ({issue}`16637`)
+* Improve performance of queries run after data was written by Trino. ({issue}`15441`)
+* Remove the `$data` system table. ({issue}`16650`)
 * Fix failure when the `$files` system table contains non-null values in the
   `key_metadata`, `split_offsets`, and `equality_ids` columns. ({issue}`16473`)
+* Fix failure when partitioned column names contain uppercase characters. ({issue}`16622`)
 
 ## Ignite connector
 
 * Add support for predicate pushdown with a `LIKE` clause. ({issue}`16396`)
 * Add support for pushdown of joins. ({issue}`16428`)
+* Add support for {doc}`/sql/delete`. ({issue}`16720`)
+
+## MariaDB connector
+
+* Fix failure when the `query` table function contains a column alias. ({issue}`16225`)
+
+## MongoDB connector
+
+* Fix incorrect results when the MongoDB `query` table function contains helper
+  functions. ({issue}`16626`)
+
+## MySQL connector
+
+* Fix failure when the `query` table function contains a column alias. ({issue}`16225`)
+
+## Oracle connector
+
+* Improve performance of queries when there is high latency to the server or
```

Comment:
"Improve performance of queries when the network latency between Trino and Oracle is high, or when selecting ..."